articles/why-python-is-good-for-machine-learning/index.md

```diff
@@ -0,0 +1,47 @@
+
+### Understanding AI, Machine Learning, and Deep Learning.
+AI refers to any intelligence shown by a machine that can lead to a feasible solution to the problem at hand. Machine learning expands on AI methods further by using [algorithms](https://www.thinkautomation.com/eli5/what-is-an-algorithm-an-in-a-nutshell-explanation/) to analyze data and learn from the same to make better decisions.
+
+Deep learning works similarly but has different capabilities, like drawing conclusions that resemble human decision-making. It is made possible by using well-structured layers of algorithms inspired by a human brain's neural network.
+
+### Reasons for Python Language for Machine Learning.
+**A huge number of libraries and frameworks:** Python language comes with many [libraries](https://data-flair.training/blogs/python-libraries/) and [frameworks](https://www.fullstackpython.com/web-frameworks.html) that make coding easy and saves on development time, making it a popular language choice among developers.
+
+The most popular libraries are [NumPy](https://pypi.org/project/numpy/), which is used for scientific calculations; [SciPy](https://pypi.org/project/scipy/) for more advanced computations; and [scikit](https://scikit-learn.org/stable/), for learning data mining and data analysis. 
+
+These libraries work alongside powerful frameworks like [TensorFlow](https://www.tensorflow.org/learn), [CNTK](https://cntk.azurewebsites.net/pythondocs/), and [Apache Spark](https://spark.apache.org/). The libraries and frameworks are essential when it comes to machine and deep learning projects.
+
+**Simplicity:** Python’s code is concise and readable even to the new developers, which is beneficial to machine and deep learning projects. Due to the language's simple syntax, the development is fast compared to many programming languages. Furthermore, it allows the developer to test algorithms without implementing them.
+
+Also, the readable code is vital for collaborative coding or even machine learning or deep learning projects; the change between development teams is easy.
+
+One can easily find a Python developer for the team as Python is a familiar platform; therefore, a new developer can quickly get acquainted with Python's concepts and work on the project instantly.
+
+The developers need not worry about the language's technicalities because Python has a simple syntax; hence they can focus more on solving a machine learning problem that requires complex algorithms.
+
+**The massive online support:** Python has been an open-source programming language and enjoys excellent support from many resources and quality documentation worldwide. Also, it has a large and active community of developers who provides their assistance and advice at any stage of development.
+
+Most scientists have adopted Python for Machine Learning and Deep Learning projects, which means most of the brightest minds worldwide, can be found in Python communities.
+
```

Comment:
`Python has a syntax that is easy to understand and friendly, with many frameworks and libraries the language offers makes development easy, fast, and productive.`

You can just say

`Python has a syntax that is easy to understand and friendly. Furthermore, the numerous frameworks and libraries boost software development.`